model/cluster_utility.go

```diff
@@ -23,6 +27,10 @@ const (
 	NginxDefaultVersion = "1.30.0"
 	// FluentbitDefaultVersion defines the default version for the Helm chart
 	FluentbitDefaultVersion = "2.8.7"
+	// CertManagerDefaultVersion defines the default version for the Helm chart
+	CertManagerDefaultVersion = "v0.13.1"
```

Comment:
Is the v in this version string intentional or a typo?

Reply:
@gigawhitlocks its intentional. Thats how they define the version for certmanager.

Reply:
Perfect, thought that might be the case.